### PR TITLE
[FEATURE] Add randomIntWithLengthInRange

### PR DESCRIPTION
**Pull request by ergenekonyigit**
## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)
- [ ] Tests

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by ergenekonyigit** [comment]
@kingdavidmartins hey, i updated `randomIntWithLengthInRange.md` 😃 👍

File: snippets/randomIntWithLengthInRange.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+### randomIntWithLengthInRange
```
Comment (Chalarangelo) [inline snippets/randomIntWithLengthInRange.md]: I suggest we name this `randomIntArrayInRange`, which is not too verbose and should be easy to find.

File: snippets/randomIntWithLengthInRange.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Returns a random integer in the specified range and length.
```
Comment (Chalarangelo) [inline snippets/randomIntWithLengthInRange.md]: Should be *Returns an array of `n` random integers in the specified range*.

File: snippets/randomIntWithLengthInRange.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Use `Array.from()` to create an empty array of the specific length. `Math.random()` to generate a random number and map it to the desired range, using `Math.floor()` to make it an integer.
+
+```js
+const randomIntWithLengthInRange = (len, min, max) =>
+	Array.from({ length: len }, () =>
+		Math.floor(Math.random() * (max - min) + min)
```
Comment (Chalarangelo) [inline snippets/randomIntWithLengthInRange.md]: As @atomiks stated, should be `min + 1`.

File: snippets/randomIntWithLengthInRange.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	);
+```
+
+<details>
```
Comment (Chalarangelo) [inline snippets/randomIntWithLengthInRange.md]: Remove the `<details>` block, it's automatically inserted by the build process.

File: snippets/randomIntWithLengthInRange.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+### randomIntWithLengthInRange
+
+Returns a random integer in the specified range and length.
+
+Use `Array.from()` to create an empty array of the specific length. `Math.random()` to generate a random number and map it to the desired range, using `Math.floor()` to make it an integer.
+
+```js
+const randomIntWithLengthInRange = (len, min, max) =>
```
Comment (Chalarangelo) [inline snippets/randomIntWithLengthInRange.md]: The footprint should be `(min, max, n = 1)`.

File: snippets/randomIntWithLengthInRange.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+	Array.from({ length: len }, () =>
+		Math.floor(Math.random() * (max - min) + min)
+	);
+```
+
+<details>
+<summary>Examples</summary>
+
+```js
+randomIntWithLengthInRange(10, 12, 35); // [ 14, 20, 12, 29, 23, 16, 26, 22, 15, 24 ]
```
Comment (Chalarangelo) [inline snippets/randomIntWithLengthInRange.md]: Example should also be updated as soon as the footprint is changed.

**Comment by ergenekonyigit** [comment]
@Chalarangelo i think it's finally done 😅

**Pull request merged**